docs/src/pages/components/cdk-components/stack/api.jsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { DxcTable } from "@dxc-technology/halstack-react";
+
+const stackPropsTable = () => {
+  return (
+    <DxcTable>
+      <tr>
+        <th>Name</th>
+        <th>Default</th>
+        <th>Description</th>
+      </tr>
+      <tr>
+        <td>align: 'start' | 'center' | 'end' | 'baseline' | 'stretch'</td>
+        <td></td>
+        <td>Alignment applied to children.</td>
+      </tr>
+      <tr>
+        <td>as: string | component</td>
+        <td>
+          <code>"div"</code>
+        </td>
+        <td>Component type or primitive that is rendered as the type of the root element.</td>
+      </tr>
+      <tr>
+        <td>children: node</td>
+        <td></td>
+        <td>Custom content inside the stack.</td>
+      </tr>
+      <tr>
+        <td>divider: boolean</td>
+        <td>
+          <code>false</code>
+        </td>
+        <td>If true a divider is shown between children.</td>
```

Comment:
I think we need to add a comma for better reading: "If true, a divider is shown between children".